test/com/ableton/PyenvTest.groovy

```diff
@@ -36,7 +36,7 @@ class PyenvTest extends BasePipelineTest {
   void assertPyenvRootInvalidRoot() {
     String pyenvRoot = '/mock/pyenv/root'
     helper.registerAllowedMethod('fileExists', [String]) { return false }
-    helper.registerAllowedMethod('isUnix', []) { return true }
+    script.env['OS'] = 'Linux'
```

Comment:
We don't have to set `script.env['OS']`, since accessing `script.env.OS` will return `null`. However, I don't feel strongly about it.

Reply:
Sure, why not. See 4d18c291a21e84cd57fbb2a014acacac507a655d